src/Commands/GenerateDocumentation.php

```diff
@@ -98,11 +98,21 @@ private function processRoutes(Generator $generator, array $routes)
         foreach ($routes as $routeItem) {
             $route = $routeItem['route'];
             /** @var Route $route */
-            if ($this->isValidRoute($route) && $this->isRouteVisibleForDocumentation($route->getAction())) {
+
+            $messageFormat = '%s route: [%s] %s';
+            $routeMethods = implode(',', $generator->getMethods($route));
+            $routePath = $generator->getUri($route);
+
+            if (!$this->isValidRoute($route) || !$this->isRouteVisibleForDocumentation($route->getAction())) {
```

Comment:
I've inverted the conditions in order to reduce nesting, since adding the try/catch looked a bit deep to me